loader/base_loader.py

```diff
@@ -104,6 +117,35 @@ def __init__(self, dss_endpoint: str, staging_bucket: str, google_project_id: st
         dss_config['DSSClient'].swagger_url = f'{self.dss_endpoint}/swagger.json'
         self.dss_client = DSSClient(config=dss_config)
 
+    def mk_s3_metadata_client(self, aws_meta_cred):
+        """Access AWS credentials from a file and supply a client for them."""
+        if not aws_meta_cred:
+            return None
+
+        client = boto3.client('sts')
+        with open(aws_meta_cred, 'r') as f:
+            rolearn = f.read().strip()  # 'arn:aws:iam::************:role/ROLE_NAME_HERE'
+
+        # 43200 seconds is 12 hours ()
+        # This setting can have a value from 900 seconds to 12 hours (as of 10.23.2018).
+        # https://docs.aws.amazon.com/cli/latest/reference/sts/assume-role.html
+        assumed_role = client.assume_role(RoleArn=rolearn, RoleSessionName='NIH-Test', DurationSeconds=43199)
```

Comment:
I think it's appropriate to have the maximum, since ideally you'd want it to be good for whatever duration you need without having to worry about it.  I don't think there'd be any unintended consequences?

If we were expecting a load to take longer than 12 hours, then I suspect we might need to have a mechanism to refresh the client.  However, I don't see that as a use-case that will come up as the json itself, I imagine, would have to be in the terabytes to even approach that.

I've made a note in the documentation though, as I agree it should be documentated and a good suggestion.